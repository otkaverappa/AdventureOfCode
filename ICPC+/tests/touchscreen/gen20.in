19
fgobzoirsmfewrpcxmow 10
xpqaciyzygudokhpypgs
enwagnedcscbjscnacma
kjgytnnuevmiklkrypwa
jyetsgitkvvuebuzqjtv
eidqtojtfgwogahziqus
npmrtgsjpmgvumnncxih
fevledmmvgeiysbratcr
hlmbxcqzzzggdbriggwb
mblmvodwiinptctsejrf
kxlqaeahkwkxzwlvkpru
zekugfmmodrbcercjrjt 10
nwsnsekevbzwhlqqqecg
juhlaapjrbfhxzwpdgva
huwohoexujgdfpofqgph
hwqevmvzsqbbnaquowuj
faokrdshlhrsdhyyutzo
ldqydivrepcmrryiwqqj
zjbeszfmsgdfjtembzgh
okthdupbkhkmqlsjmxxe
gampvqcwrkehuzoytdzg
nkufxmokmoqsqchmulkl
vosqqgqlmrrbdlgdaxnm 10
lfedilpczaquqimgresf
wlgbzmezjrnwzraherld
rdaimmqftkmpwssxgxws
qjqpcqyikmobpoldceiv
owmmofjxefruqhmuakdk
ytnohbrlfcjtyvgpcrog
xfdpopmobraakootphew
lnpmlxdnprwoyzfoqtdr
kftwvjpkrwielyqwxvmo
okeolmdbfiuqnomjxevq
aevlenkdjysziyotkrwq 10
zqiogwxgcvwcbtqfjciu
actidhbnaxfzppqwmneq
katlwlshnbbnfwxigayg
zdiovymhnqzatslrfeyu
hzjovgzdgxkfcuwxsigh
bghuatlizjcikmwhuvld
uxkxrgwjrdsslaolvbvw
myewmcfiyslupytgeqrx
vkrgmfthgrfspjqdlyll
qzigxbmdrgboquweasli
jsdybvdotockpkqmneqh 10
ktvcnsgpmtyvldvoyyfs
ohcetusiakpldlptfykt
tkofnjvmjagzjideewne
gcrmphhwhrpacfhpqddz
fjzrrevxckcjptxecedj
vukxatpqyssectxvzstc
fxluqiatodejaqjakysi
qmouhlqhgljliwhaghvu
nadnsmnckhndubzbppix
ctkmprnvaipnktadhphu
zuztyywnqhmsawfsnunn 10
efdpaduhvbdwycrwbnlr
wxlztsrimgvtnaineexb
haxffodgdozankbiduqr
aokoouduycxgdulikpqq
fpstztdenuwpkgfybjvb
nsjqpxazospthhojcsnp
olhysmyvxtylmidbfgct
ysoibftgxixnuenotljq
fjdtrjuypwrpqizunsam
bxzxcolvzwngftzzctxr
sriizicoccafcbceppbo 10
lqvtjuslqreiinqjvvxz
zxfbahhpxkgkbdfkzzyp
rczbrqloniomitoicvab
iiojltvktvckabnrtyig
jwttrjbvgdzolnxwgvjb
qlnqobkizsqkqjdjuhgb
kfpyspwbmfccsqvhrhrs
bhdrtgcnnioaogyixujj
amnuckdvrunscqlxzpkm
abmqhmyghipjwedapjvi
dkdhbqeairoisdybpwhy 10
gwjecperyzcdkflnxrpf
ifncildakkasikxlzbcz
dgcpnnelhvqragultxnf
knyszxdaaizfocwepapy
xhqynmjhlyovlmqnltnm
dnrupoygqnfowxokjxtu
xjskwixjdlvhypbndbvv
qajmxxyiwtftdxgbhdkk
ogrovudavzvlzhazgyjf
soaxngyukkfaqypntspq
tncvuewccfhwwitjotey 10
elzwjqmckbveqxbmdxpf
cweygxkvtovxbwvkmhpz
lkfbjgomddthbxiixsfq
iapjxktlukkfurifxwrc
bnkfkuphmuzuuoftbyhv
jrcflklijelmrxtcrilf
clbzbgucgbxrvzxgmkov
qbjkadotlzyqmzpohlqo
nqfipeodoeahikrinfca
ecqrchhjtxzgnfqfjgia
mjjuvafifjkllbepklbd 10
jalyhddtjovwzesuexej
iowwrblcoohxrvyyybti
rogqsymwyqhiheeahscv
gmvzhtafwtpoivgawtxw
lgesmlttdxpllmlsfnad
grropyrlrojcwpxjascf
rttdhgxmtxqzpjqehhra
xcftrccuwhzncsqkyoyt
nqtecjklqbmpftixxntt
wugynxknnlhabchdlrpd
vduawcztrsmnnuocrarg 10
lajncqspjjuemohktgfk
braqmqufqlocmxroplgy
vafjpmwiubuxtwnhmjof
vcjjbczqnfrkgwwxishe
tbcoyrxmboryqahucims
odcwbavlscqoducdlcqm
qjmiktcocpgsulowllhg
nzustywfaotsxgbjbfaf
ugxrtnnhywplwlepjcwm
qqepyhbznbfjjeafspms
odemokbzoalgssvqbyro 10
bwalbaqvsenihtuxfxwt
xkzpcxhdxzuzxukawcvq
ilyrguomumhrwjjaisef
raepwqpsunlealvgglua
yetwnexvxdcodhfzzxtu
keamsyuyjpajvtikafhx
ijnnstosqhocoooimigy
zijudredwlagxotpjhjz
rxbhoppczxazfmvidaoc
nqikgecpnloelrnziebi
bejiqetvehxuxigemivz 10
wkghdtimajwbpfmhlfdq
ocmnkstydpyzbeigzquz
bqdrxryjydbmipbsjxrm
mrnpvvvwnqypibgisgts
mweulhmuefjswyhtwfqj
xoyhrgplnkdzgjttqhqx
pbplzxhxezhbnfigmarb
mwcthxoxeewvfmhhlqep
pnscvakjcdkpzmkgjyfp
fclmquvbkcszpkdkmpvp
shgttsbeqiwxmjkcdfdp 10
hvozhtluihlcotvklyqb
gmbtvlvasbpcyfbgaock
xnmmgjyrhouodvjaigdb
hsdgagoawqlufzgljges
wygzwrceyfffzknzsdcp
vnjaorlxzpsvqaxmszqq
eyxdinfdshsowdokwakv
rcqhcntwmkmskmysbdvt
mphiuwuqyglpkczprulg
gayqmyjpdgipyqzsowjo
cwdpyeeszryfrwwfxhua 10
pfqnxrilntzspejqlnik
ggryepfbyccohudellrb
esvwyemjruvxcpxgefkf
hotqkywvmnyshtohxatq
woqyfnhmutrbhmtrkpow
eprliftgimyecofkcowy
hncqbvjllzkpobdxiyfq
kdxoscywqwuxjypkuyyf
xkwmnzlvxqojvlappanf
xkegittcrtjodgcsfnpf
fdqdqqsfrikosowajrfa 10
korputhcjxjqczttqoyj
wlzqzvqjpxlbmcrgyakj
xtzctvvlltwhgvzgsprh
pejbjajjcuscnteiqzvb
urkbmjjfbcmqgxrrzaac
wuemoiuejqgfhsiuctbd
vptenkvmmwojquvgernn
hvvrndnpxosudnasxwfm
uvxmssvwjimtdhmqmzgl
paiuoinneubarypjqkgc
suxvbjnpkvdzwltmtiba 10
cdcwbrfubnythvqkggaq
bdrzqknjvplxuovxfbrh
orcymviubkmepdgfqtrl
kckgqgeyhvhxnjvbggyj
qkqgownererejbkbhozq
mgqbsleatckknasdyhjr
mavxbgylwybkgtlafqba
ulkjneolmxcaxaxbivme
vppejcequfqpsdafjpsv
mxxmxunfsblpqavbfarz
hkqanrhygauuztgyqvdi 10
wozprvqwxixgsoghhqfp
sbjrvsplnuvmkwbdttbs
bzzvphewxklpnuikozxe
vssgrtjkmndqmcndlszj
ekatgjevkbbgwtonozzc
mcsagidsacdgodzvodsb
fvhdovqcurejwzjehnyj
rbpghodxuxybsheiewkb
nqkjrtqaiojasyizplyj
kzkcgqknmwoboyngtdge
tselsokhblslkepsuzfi 10
xvmltbrpgatzszlknxuq
iobstqlprsaoombjpvav
vtupufbjdvznkcgfstul
nwbbkdnzynuvjrmfwnpb
jqpvtxanqwzftajffwff
lbawsmcpctsnjhieeksw
itcdvnjbjqgwrjtkxxbz
stoddyijicitvmyszhtk
zcgrnbdmagosbcwebgql
iyefkfxlmtvoxbfmfkyf
