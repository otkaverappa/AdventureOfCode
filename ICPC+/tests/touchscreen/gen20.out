jyetsgitkvvuebuzqjtv 56
mblmvodwiinptctsejrf 64
eidqtojtfgwogahziqus 76
enwagnedcscbjscnacma 80
fevledmmvgeiysbratcr 80
hlmbxcqzzzggdbriggwb 80
npmrtgsjpmgvumnncxih 82
kxlqaeahkwkxzwlvkpru 83
kjgytnnuevmiklkrypwa 85
xpqaciyzygudokhpypgs 88
faokrdshlhrsdhyyutzo 55
zjbeszfmsgdfjtembzgh 62
juhlaapjrbfhxzwpdgva 71
hwqevmvzsqbbnaquowuj 72
nkufxmokmoqsqchmulkl 75
gampvqcwrkehuzoytdzg 76
huwohoexujgdfpofqgph 79
okthdupbkhkmqlsjmxxe 79
nwsnsekevbzwhlqqqecg 80
ldqydivrepcmrryiwqqj 85
wlgbzmezjrnwzraherld 72
qjqpcqyikmobpoldceiv 73
kftwvjpkrwielyqwxvmo 74
ytnohbrlfcjtyvgpcrog 77
okeolmdbfiuqnomjxevq 83
rdaimmqftkmpwssxgxws 84
lfedilpczaquqimgresf 87
owmmofjxefruqhmuakdk 90
xfdpopmobraakootphew 93
lnpmlxdnprwoyzfoqtdr 97
zdiovymhnqzatslrfeyu 54
actidhbnaxfzppqwmneq 55
zqiogwxgcvwcbtqfjciu 72
uxkxrgwjrdsslaolvbvw 73
bghuatlizjcikmwhuvld 75
vkrgmfthgrfspjqdlyll 75
katlwlshnbbnfwxigayg 78
myewmcfiyslupytgeqrx 83
qzigxbmdrgboquweasli 83
hzjovgzdgxkfcuwxsigh 85
fjzrrevxckcjptxecedj 57
ktvcnsgpmtyvldvoyyfs 66
ohcetusiakpldlptfykt 75
nadnsmnckhndubzbppix 76
qmouhlqhgljliwhaghvu 79
ctkmprnvaipnktadhphu 84
tkofnjvmjagzjideewne 84
fxluqiatodejaqjakysi 86
gcrmphhwhrpacfhpqddz 93
vukxatpqyssectxvzstc 96
efdpaduhvbdwycrwbnlr 68
bxzxcolvzwngftzzctxr 70
fpstztdenuwpkgfybjvb 70
fjdtrjuypwrpqizunsam 77
haxffodgdozankbiduqr 77
olhysmyvxtylmidbfgct 79
aokoouduycxgdulikpqq 80
wxlztsrimgvtnaineexb 85
ysoibftgxixnuenotljq 87
nsjqpxazospthhojcsnp 96
iiojltvktvckabnrtyig 67
kfpyspwbmfccsqvhrhrs 70
jwttrjbvgdzolnxwgvjb 73
abmqhmyghipjwedapjvi 74
amnuckdvrunscqlxzpkm 74
qlnqobkizsqkqjdjuhgb 78
zxfbahhpxkgkbdfkzzyp 79
lqvtjuslqreiinqjvvxz 91
bhdrtgcnnioaogyixujj 92
rczbrqloniomitoicvab 112
dgcpnnelhvqragultxnf 62
xhqynmjhlyovlmqnltnm 64
dnrupoygqnfowxokjxtu 66
qajmxxyiwtftdxgbhdkk 66
ogrovudavzvlzhazgyjf 73
soaxngyukkfaqypntspq 73
knyszxdaaizfocwepapy 75
gwjecperyzcdkflnxrpf 85
xjskwixjdlvhypbndbvv 86
ifncildakkasikxlzbcz 105
elzwjqmckbveqxbmdxpf 68
bnkfkuphmuzuuoftbyhv 72
lkfbjgomddthbxiixsfq 73
nqfipeodoeahikrinfca 74
clbzbgucgbxrvzxgmkov 75
cweygxkvtovxbwvkmhpz 80
ecqrchhjtxzgnfqfjgia 81
jrcflklijelmrxtcrilf 82
qbjkadotlzyqmzpohlqo 85
iapjxktlukkfurifxwrc 93
xcftrccuwhzncsqkyoyt 65
wugynxknnlhabchdlrpd 71
lgesmlttdxpllmlsfnad 73
gmvzhtafwtpoivgawtxw 75
nqtecjklqbmpftixxntt 77
jalyhddtjovwzesuexej 78
rttdhgxmtxqzpjqehhra 78
rogqsymwyqhiheeahscv 79
grropyrlrojcwpxjascf 81
iowwrblcoohxrvyyybti 85
vcjjbczqnfrkgwwxishe 57
lajncqspjjuemohktgfk 67
nzustywfaotsxgbjbfaf 69
odcwbavlscqoducdlcqm 77
vafjpmwiubuxtwnhmjof 77
ugxrtnnhywplwlepjcwm 79
qjmiktcocpgsulowllhg 82
braqmqufqlocmxroplgy 88
qqepyhbznbfjjeafspms 92
tbcoyrxmboryqahucims 94
yetwnexvxdcodhfzzxtu 74
ilyrguomumhrwjjaisef 76
raepwqpsunlealvgglua 77
nqikgecpnloelrnziebi 82
bwalbaqvsenihtuxfxwt 85
ijnnstosqhocoooimigy 87
rxbhoppczxazfmvidaoc 87
keamsyuyjpajvtikafhx 92
xkzpcxhdxzuzxukawcvq 94
zijudredwlagxotpjhjz 104
bqdrxryjydbmipbsjxrm 60
fclmquvbkcszpkdkmpvp 68
pbplzxhxezhbnfigmarb 71
wkghdtimajwbpfmhlfdq 71
mwcthxoxeewvfmhhlqep 72
xoyhrgplnkdzgjttqhqx 74
mrnpvvvwnqypibgisgts 75
mweulhmuefjswyhtwfqj 76
ocmnkstydpyzbeigzquz 80
pnscvakjcdkpzmkgjyfp 80
wygzwrceyfffzknzsdcp 47
gmbtvlvasbpcyfbgaock 61
rcqhcntwmkmskmysbdvt 62
xnmmgjyrhouodvjaigdb 67
eyxdinfdshsowdokwakv 73
vnjaorlxzpsvqaxmszqq 75
hsdgagoawqlufzgljges 87
hvozhtluihlcotvklyqb 89
gayqmyjpdgipyqzsowjo 91
mphiuwuqyglpkczprulg 93
xkegittcrtjodgcsfnpf 58
kdxoscywqwuxjypkuyyf 62
esvwyemjruvxcpxgefkf 63
hotqkywvmnyshtohxatq 68
eprliftgimyecofkcowy 72
woqyfnhmutrbhmtrkpow 78
ggryepfbyccohudellrb 81
xkwmnzlvxqojvlappanf 81
pfqnxrilntzspejqlnik 89
hncqbvjllzkpobdxiyfq 91
uvxmssvwjimtdhmqmzgl 62
hvvrndnpxosudnasxwfm 70
vptenkvmmwojquvgernn 78
xtzctvvlltwhgvzgsprh 80
wlzqzvqjpxlbmcrgyakj 83
pejbjajjcuscnteiqzvb 88
urkbmjjfbcmqgxrrzaac 92
wuemoiuejqgfhsiuctbd 93
paiuoinneubarypjqkgc 95
korputhcjxjqczttqoyj 100
cdcwbrfubnythvqkggaq 66
qkqgownererejbkbhozq 67
ulkjneolmxcaxaxbivme 70
kckgqgeyhvhxnjvbggyj 71
mavxbgylwybkgtlafqba 75
bdrzqknjvplxuovxfbrh 76
mxxmxunfsblpqavbfarz 79
orcymviubkmepdgfqtrl 84
mgqbsleatckknasdyhjr 86
vppejcequfqpsdafjpsv 87
kzkcgqknmwoboyngtdge 69
ekatgjevkbbgwtonozzc 70
rbpghodxuxybsheiewkb 70
fvhdovqcurejwzjehnyj 73
mcsagidsacdgodzvodsb 77
vssgrtjkmndqmcndlszj 77
wozprvqwxixgsoghhqfp 78
bzzvphewxklpnuikozxe 82
sbjrvsplnuvmkwbdttbs 82
nqkjrtqaiojasyizplyj 96
vtupufbjdvznkcgfstul 73
lbawsmcpctsnjhieeksw 75
stoddyijicitvmyszhtk 82
jqpvtxanqwzftajffwff 85
nwbbkdnzynuvjrmfwnpb 85
iobstqlprsaoombjpvav 87
itcdvnjbjqgwrjtkxxbz 89
xvmltbrpgatzszlknxuq 89
iyefkfxlmtvoxbfmfkyf 90
zcgrnbdmagosbcwebgql 90
